src/component/common/ConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/ConstraintAccordionEditBody.styles.ts

```diff
@@ -3,6 +3,7 @@ import { makeStyles } from 'tss-react/mui';
 export const useStyles = makeStyles()(theme => ({
     inputContainer: {
         padding: '1rem',
+        backgroundColor: theme.palette.grey[100],
```

Comment:
Using the pallette colors and shades directly in components makes it very hard to theme. I discovered this when making the dark mode PR. Because in dark mode we switch out the colors, we would have to override the grey palette with something that isn't grey. 

It would be better to create more general abstractions in the theme types and use this in the components, because then we could easier theme it by swapping out the general abstraction. I've fixed all of these up in the dark mode PR.